src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -91,7 +94,7 @@ private BoundExpression BindSwitchExpression(SwitchExpressionSyntax node, Diagno
             Binder originalBinder,
             DiagnosticBag diagnostics)
         {
-            return this.Next.BindSwitchExpressionCore(node, originalBinder, diagnostics);
+            return this.Next!.BindSwitchExpressionCore(node, originalBinder, diagnostics);
```

Comment:
>Next!. [](start = 24, length = 6)

In other places, we've used an assertion instead of a suppression, because this is not the case of a compiler limitation, but rather remote knowledge.
See guidelines: [https://github.com/dotnet/roslyn/blob/master/docs/contributing/Nullable%20Annotations.md](https://github.com/dotnet/roslyn/blob/master/docs/contributing/Nullable%20Annotations.md)